server/src/main/java/org/apache/iotdb/db/metadata/VectorPartialPath.java

```diff
@@ -111,6 +155,15 @@ public int hashCode() {
     return Objects.hash(super.hashCode(), subSensorsList);
   }
 
+  @Override
+  public PartialPath getExactPath() {
```

Comment:
what is this for?

Reply:
If the partialPath is AlignedPath and it has only one measurement, return the measurement's PartialPath. Otherwise, return this partialPath.